src/Message/CaptureResponse.php

```diff
@@ -45,6 +45,7 @@ public function getRedirectUrl()
 
 	public function getRedirectMethod()
 	{
+		// FIXME: A redirect should nearly always be a GET. Is this really correct?
 		return 'POST';
```

Comment:
In the very early days, yes a "redirect" is a GET. That's not so clear cut now with a lot of gateways. Examples include:

* A HTTP redirect (a GET). Can only carry GET parameters with it, so has limited length and may be cached in proxies.
* A POST redirect - a HTML POST form with an automatic submit. If the source page is HTTPS then the destination page must also be HTTPS to avoid browser errors.
* A JavaScript redirect - not a form but a GET redirect using JavaScript. This skips browser errors when jumping from a HTTPS page to a HTTP page.